src/vcastTestInterface.ts

```diff
@@ -722,12 +727,13 @@ function createScriptTemplate(testNode: testNodeType): string {
   return scriptTemplateLines.join("\n");
 }
 
-export async function newTestScript(nodeID: string) {
+
+
+export async function newTestScript(testNode: testNodeType) {
```

Comment:
Very minor, but I am not sure why we added spaces here (I'm also surprised `prettier` didn't remove them).